[2023-05-18T17:43:51.283+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: Anomaly_Detection_Pipeline.predict_anomaly manual__2023-05-18T17:41:27.835086+00:00 [queued]>
[2023-05-18T17:43:51.347+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: Anomaly_Detection_Pipeline.predict_anomaly manual__2023-05-18T17:41:27.835086+00:00 [queued]>
[2023-05-18T17:43:51.351+0000] {taskinstance.py:1279} INFO - 
--------------------------------------------------------------------------------
[2023-05-18T17:43:51.355+0000] {taskinstance.py:1280} INFO - Starting attempt 1 of 1
[2023-05-18T17:43:51.359+0000] {taskinstance.py:1281} INFO - 
--------------------------------------------------------------------------------
[2023-05-18T17:43:51.472+0000] {taskinstance.py:1300} INFO - Executing <Task(PythonOperator): predict_anomaly> on 2023-05-18 17:41:27.835086+00:00
[2023-05-18T17:43:51.494+0000] {standard_task_runner.py:55} INFO - Started process 583 to run task
[2023-05-18T17:43:51.519+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'Anomaly_Detection_Pipeline', 'predict_anomaly', 'manual__2023-05-18T17:41:27.835086+00:00', '--job-id', '82', '--raw', '--subdir', 'DAGS_FOLDER/anomaly.py', '--cfg-path', '/tmp/tmph4a4cnx5']
[2023-05-18T17:43:51.524+0000] {standard_task_runner.py:83} INFO - Job 82: Subtask predict_anomaly
[2023-05-18T17:43:51.999+0000] {task_command.py:388} INFO - Running <TaskInstance: Anomaly_Detection_Pipeline.predict_anomaly manual__2023-05-18T17:41:27.835086+00:00 [running]> on host 0ea9fc9e859a
[2023-05-18T17:43:52.541+0000] {taskinstance.py:1509} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=dataength
AIRFLOW_CTX_DAG_ID=Anomaly_Detection_Pipeline
AIRFLOW_CTX_TASK_ID=predict_anomaly
AIRFLOW_CTX_EXECUTION_DATE=2023-05-18T17:41:27.835086+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-05-18T17:41:27.835086+00:00
[2023-05-18T17:43:55.877+0000] {logging_mixin.py:137} INFO - Response data: {'predictions': [0, 0, 0, 0, 0, 0, 0, 0, -1, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0]}
[2023-05-18T17:43:55.942+0000] {python.py:177} INFO - Done. Returned value was: 200
[2023-05-18T17:43:56.177+0000] {taskinstance.py:1323} INFO - Marking task as SUCCESS. dag_id=Anomaly_Detection_Pipeline, task_id=predict_anomaly, execution_date=20230518T174127, start_date=20230518T174351, end_date=20230518T174356
[2023-05-18T17:43:56.283+0000] {local_task_job.py:208} INFO - Task exited with return code 0
[2023-05-18T17:43:56.369+0000] {taskinstance.py:2578} INFO - 0 downstream tasks scheduled from follow-on schedule check
